scss/bootstrap/overrides/style/_thumbnail.scss

```diff
@@ -0,0 +1,75 @@
+.crm-mosaico-template-item {
```

Comment:
Its already sorted. Current plugins sort @include based on the attribute inside it. Is it incorrect?